src/compute/src/logging/compute.rs

```diff
@@ -258,7 +258,7 @@ pub fn construct<A: Allocate>(
                                         Datum::Int64(logical as i64),
                                     ]),
                                     time_ms,
-                                    delta,
+                                    delta as i64,
```

Comment:
Nit: Let's use (`From<i8>`)[https://doc.rust-lang.org/std/primitive.i64.html#impl-From%3Ci8%3E] here to make sure it's a lossless conversion.